src/main.js

```diff
@@ -120,17 +123,4 @@ function generateRandomPoster() {
   posterQuote.innerText = quotes[getRandomIndex(quotes)];
```

Comment:
WIP: Incorporate a 'for' loop to DRY up our getRandomIndex() function calls. 
var posterData = [images, titles, quotes]